src/js/devtools/App.jsx

```diff
@@ -16,25 +16,28 @@
 
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
-import PropTypes from 'prop-types';
-import FragmentList from '../components/FragmentList/fragmentList';
+// import PropTypes from 'prop-types';
```

Comment:
Why those are commented?

Reply:
It was just for testing purposes, I commited it this way by mistake